src/Player.as

```diff
@@ -172,6 +174,10 @@ package {
         }
       }
 
+      if (iconfig.step !== undefined) {
```

Comment:
add this to the documentation along with what it means to set the player in stepping mode